crates/driver/src/domain/competition/solution/settlement.rs

```diff
@@ -215,6 +215,20 @@ impl Settlement {
             .into()
     }
 
+    fn scoring(
```

Comment:
Can you create a task to capture the work of combining the `score()` and `scoring()` method once CIP38 is live and no longer at risk of getting reverted?

Also, I know I was against this before, but I feel now it makes sense to call this what it is
```suggestion
    fn cip38_score(
```

Reply:
renamed